Apps/WebClient/src/Properties/launchSettings.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Looks like this file is only needed by the Visual Studio. Not sure if we should keep it in the repo.